### PR TITLE
Prototype 3

### PR DESCRIPTION
**Pull request by ajm27pdx**
Moving p3 validation into main branch

### CODE REVIEW

File: prototyping/prototype-3/p3-server.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        command_list[0] = a #adding the command to a list
+        print("CMD",command_list)
+    elif o in ("-i","--ID"):
+        ID = a #for PB number
+        PB_ID = a
+        print("PB ID: ",PB_ID[0])
+    elif o in ("-o", "--output"):
+        output = a
+    else:
+        assert False, "unhandled option"
```
Comment (ryanpdx) [inline prototyping/prototype-3/p3-server.py:47]: Use argparse for this

File: prototyping/prototype-2/server.conf
Comment (ryanpdx) [inline prototyping/prototype-2/server.conf:1]: I would be better to use standard configure file format like `.conf`.

**Review by ryanpdx — CHANGES_REQUESTED** [review]
This is a little hard to review as there is no documentation.

Please fill put the README.md with high level overview of how it work, how to setup environment to for the project, and how to run the project. Also recommend adding mermaid diagram to the readme to show data flow between hardware and software as well as the server/client dataflow.

Please add docstring to all functions.

Also add `pyproject.toml` (has all the metadata for the project) and `requirements.txt` (used by developers to quickly install all dev dependencies) files. See the oresat-c3-software repo for examples.

What are 3 multiple prototype directories? There should be one source directory, called something like `oresat_solar_simulator`.

File: prototyping/prototype-3/p3-server.py (hunk 1 above)
Comment (ajm27pdx) [reply prototyping/prototype-3/p3-server.py:47]: Done! Much easier.

File: prototyping/prototype-2/server.conf
Comment (ajm27pdx) [reply prototyping/prototype-2/server.conf:1]: This was a bit of a test implementation, I've switched to JSON conf files for the current version.

**Comment by ajm27pdx** [comment]
Added `requirements.txt` and will finish building out `pyproject.toml` for final version clean up. Prototype folders were only for testing and capstone stuff, we plan to remove for the 'release' version.

**Pull request merged**